xc/device.ml

```diff
@@ -1409,29 +1409,46 @@ module Vkbd = struct
 end
 
 module Vusb = struct
-  let call_usb_reset hostbus hostport =
-    let usb_reset_script = "/opt/xensource/libexec/usb_reset.py" in
+  let exec_usb_reset_script argv =
+    let stdout, stderr = Forkhelpers.execute_command_get_output !Xc_resources.usb_reset_script argv in
+    debug "usb_reset_script %s returned stdout=%s stderr=%s" (String.concat " " argv) stdout stderr
+
+  let cleanup domid =
+    try
+      let cmd = ["cleanup"; "-d"; string_of_int domid] in
+      exec_usb_reset_script cmd
+    with _ ->
```

Comment:
Let's catch the error name and add it to the warning for now:
```ocaml
with err ->
   warn "Failed to clean up VM %s: %s" (string_of_int domid) (Printexc.to_string err)
```
If we see that it is useless for debugging purposes we can always change it later.